ngraph/core/reference/include/ngraph/runtime/reference/experimental_detectron_prior_grid_generator.hpp

```diff
@@ -0,0 +1,56 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <algorithm>
```

Comment:
I can't see any algorithm, array, assert, math, functional, map,... here. Are this includes required here?

To many `includes` will extend compilation time.

Reply:
This header file is under construction, and this PR is in the status 'draft'.